src/components/Badge/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+import { BaseProps } from '../types';
+import { StyledBadge, StyledBadgeContainer } from './styled/badge';
+
+type badgeVariant = 'default' | 'lightest' | 'outline-brand' | 'brand';
+
+interface Props extends BaseProps {
+    label?: ReactNode;
+    variant?: badgeVariant;
+    style?: object;
+}
+
+const Badge: React.FC<Props> = props => {
+    const { label, variant, style } = props;
+    return (
+        <StyledBadgeContainer>
+            <StyledBadge style={style} variant={variant}>
+                {label}
+            </StyledBadge>
```

Comment:
We need to make a subcomponent that check if label is a string because if the label is a node with a View we can't wrap the label with a Text since react native will not render the content when a View is inside a Text. Look at title file in Card component.